### PR TITLE
Support for size variants - better highlight on hover / navigation

### PR DESCRIPTION
**Pull request by fraincs**
Issue:

- [Change Dropdown color bar behaviour](https://github.com/gsoft-inc/sg-orbit/issues/160)
- [Dropdown selection list should be the same size of it's parent](https://github.com/gsoft-inc/sg-orbit/issues/183)
- [Support for large dropdowns](https://github.com/gsoft-inc/sg-orbit/issues/172)

## What I did

- CSS changes in order to accommodate a large dropdown variant.
- CSS to change height of dropdown selection in order to follow the same height as it's parent/trigger
- Added a small avatar in complex large dropdowns (see https://react.semantic-ui.com/elements/image/) a better way to deal with this would be to fix this [issue](https://github.com/gsoft-inc/sg-orbit/issues/199)
- CSS to support hovered but not selected option

## How to test

Documentation has been updated.
Chromatic tests are updated and split in two chromatic tests default and complex

### CODE REVIEW

File: storybook/stories/semantic-ui/react/dropdown/standard.stories.mdx [hunk 1]
```diff
@@ -256,14 +256,15 @@ A dropdown can take the width of its container.
 
 A dropdown can vary in size.
 
-Available sizes are: "small" and "medium".
+Available sizes are: `small`, `medium` and `large`.
```
Comment (patricklafrance) [inline storybook/stories/semantic-ui/react/dropdown/standard.stories.mdx]: Change for:

"Can be set to: "small", "large" or omitted." to be consistant with the other pages.

image

**Comment by patricklafrance** [comment]
Can you create issues with the next steps please. My guess is something like this:

Create a custom dropdown HOC that will:

- Support a `size` prop
- Customize the default behavior of the dropdown `selectOnBlur={false}` & `selectOnNavigation={false}`

**Comment by patricklafrance** [comment]
Also please accept the chromatic changes.

**Comment by fraincs** [comment]
> Can you create issues with the next steps please. My guess is something like this:
> 
> Create a custom dropdown HOC that will:
> 
> * Support a `size` prop
> * Customize the default behavior of the dropdown `selectOnBlur={false}` & `selectOnNavigation={false}`

There's already an issue for the default behavior HOC should't we expand on this one?

**Comment by patricklafrance** [comment]
> > Can you create issues with the next steps please. My guess is something like this:
> > Create a custom dropdown HOC that will:
> > 
> > * Support a `size` prop
> > * Customize the default behavior of the dropdown `selectOnBlur={false}` & `selectOnNavigation={false}`
> 
> There's already an [issue](https://github.com/gsoft-inc/sg-orbit/issues/184) for the default behavior HOC should't we expand on this one?

Ok forget what I wrote then!

**Pull request merged**